glide.lock

```diff
@@ -40,6 +40,8 @@ imports:
   version: 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
 - name: github.com/JiangtianLi/gettext
   version: a8983c062be4b565d723c478922d7736e04fdba4
+- name: github.com/kelseyhightower/envconfig
```

Comment:
the only way to do that is to pin every dep to a sha. so that they never move. As it stands we have some dependencies with patch level constraints which means they could get updated if the glide.lock file needs to be rebuilt (such as adding a new dep).